src/Sulu/Bundle/WebsiteBundle/DependencyInjection/Configuration.php

```diff
@@ -56,6 +56,13 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('sitemap')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->scalarNode('dump_dir')->defaultValue('%sulu.cache_dir%/sitemaps')->end()
+                    ->scalarNode('default_host')->defaultNull()->end()
```

Comment:
The problem I see here is that if we need it also in another place in the future we'll have a BC break here, which can be solved in different ways:

* We fill this value with a `PreprendExtension`, which means more code than necessary
* We just remove this configuration value and accept the BC break
* We duplicate the value in the configuration

None of this options are really satisfying in my opinion, so I would take a few minutes to decide where it belongs if we have multiple bundles dependent on this value.